translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+msgstr ""
+"\n"
+"\t\tCria uma role com uma única regra."
+
+#: pkg/kubectl/cmd/create_secret.go:47
+msgid ""
+"\n"
+"\t\tCreate a secret based on a file, directory, or specified literal value.\n"
+"\n"
+"\t\tA single secret may package one or more key/value pairs.\n"
+"\n"
+"\t\tWhen creating a secret based on a file, the key will default to the "
+"basename of the file, and the value will\n"
+"\t\tdefault to the file content.  If the basename is an invalid key, you may "
+"specify an alternate key.\n"
+"\n"
+"\t\tWhen creating a secret based on a directory, each file whose basename is "
+"a valid key in the directory will be\n"
+"\t\tpackaged into the secret.  Any directory entries except regular files "
+"are ignored (e.g. subdirectories,\n"
+"\t\tsymlinks, devices, pipes, etc)."
+msgstr ""
+"\n"
+"\t\tCria um secret com base em um arquivo, diretório, ou um valor literal "
+"especificado.\n"
+"\n"
+"\t\tUm secret único pode conter um ou mais pares de chave/valor.\n"
+"\n"
+"\t\tQuando criar um secret com base em um arquivo, a chave será por padrão o "
+"nome do arquivo, e o valor será\n"
+"\t\tpor padrão o conteúdo do arquivo.  Se o nome do arquivo for uma chave "
+"inválida, você deve especificar uma chave alternativa.\n"
+"\n"
+"\t\tQuando criar um secret com base em um diretório, cada arquivo cujo o "
+"nome é uma chave válida no diretório será\n"
+"\t\tcolocada no configmap.  Qualquer entrada de diretório, exceto as com "
+"arquivos válidos serão ignorados (por exemplo: sub-diretórios,\n"
+"\t\tsymlinks, devices, pipes, etc)."
+
+#: pkg/kubectl/cmd/create_serviceaccount.go:32
+msgid ""
+"\n"
+"\t\tCreate a service account with the specified name."
+msgstr ""
+"\n"
+"\t\tCria uma conta de serviço com um nome especificado."
+
+#: pkg/kubectl/cmd/run.go:52
+msgid ""
+"\n"
+"\t\tCreate and run a particular image, possibly replicated.\n"
+"\n"
+"\t\tCreates a deployment or job to manage the created container(s)."
+msgstr ""
+"\n"
+"\t\tCria e executa uma imagem específica, possivelmente replicada.\n"
+"\n"
+"\t\tCria um deployment ou job para gerenciar o(s) container(s) criado(s)."
+
+#: pkg/kubectl/cmd/autoscale.go:34
+msgid ""
+"\n"
+"\t\tCreates an autoscaler that automatically chooses and sets the number of "
+"pods that run in a kubernetes cluster.\n"
+"\n"
+"\t\tLooks up a Deployment, ReplicaSet, or ReplicationController by name and "
+"creates an autoscaler that uses the given resource as a reference.\n"
+"\t\tAn autoscaler can automatically increase or decrease number of pods "
+"deployed within the system as needed."
+msgstr ""
+"\n"
+"\t\tCria um autoscaler que automaticamente escolhe e configura quantos pode "
+"irão executar em um kubernetes cluster.\n"
+"\n"
+"\t\tProcura por um Deployment, ReplicaSet, ou ReplicationController por nome "
+"e cria um autoscaler que utiliza o recurso fornecido como referência.\n"
+"\t\tUm autoscaler pode automaticamente aumentar ou reduzir o número de pods "
+"quando necessário."
```

Comment:
I'm missing the actual command here. Don't know if they are part of the original string, or if this has changed with time.

Reply:
all good